code/modules/mob/living/carbon/alien/humanoid/queen.dm

```diff
@@ -23,6 +23,18 @@
 	health = 400
 	icon_state = "alienq"
 
+/mob/living/carbon/alien/humanoid/royal/queen/maid
+	name = "alien queen maid"
+	caste = "qmaid"
+	maxHealth = 400
+	health = 400
+	icon_state = "alienqmaid"

```

Comment:
Shouldn't be a reason to have any differences with the subtype other than the icon state - that, or I don't know xeno code well enough